c_glib/arrow-dataset-glib/scanner.cpp

```diff
@@ -48,63 +48,47 @@ enum {
   PROP_SCANNER = 1,
 };
 
-G_DEFINE_TYPE_WITH_PRIVATE(GADatasetScanner,
-                           gadataset_scanner,
-                           G_TYPE_OBJECT)
+G_DEFINE_TYPE_WITH_PRIVATE(GADatasetScanner, gadataset_scanner, G_TYPE_OBJECT)
 
-#define GADATASET_SCANNER_GET_PRIVATE(obj)        \
-  static_cast<GADatasetScannerPrivate *>(         \
-    gadataset_scanner_get_instance_private(       \
-      GADATASET_SCANNER(obj)))
+#define GADATASET_SCANNER_GET_PRIVATE(obj) \
+  static_cast<GADatasetScannerPrivate*>(   \
+      gadataset_scanner_get_instance_private(GADATASET_SCANNER(obj)))
```

Comment:
Did this get autoformatted somehow? (Not sure if there's an expected style for the GLib bindings though.)

Reply:
Please don't format `c_glib/` files automatically.
`c_glib/` doesn't use auto format tool.
Please keep the current style as much as possible.

Reply:
Thank you for catching this.  My IDE was overzealous.  I'll revert back to the original formatting.